### PR TITLE
Remove extra symbols

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### CODE REVIEW

File: promql/test.go [hunk 1]
```diff
@@ -461,7 +461,7 @@ func (t *Test) exec(tc testCommand) error {
 		}
 
 		// Check query returns same result in range mode,
```
Comment (cstyan) [inline promql/test.go]: I think the `,` at the end could be removed as well.

Comment (yuxiaobo96) [reply promql/test.go]: Sorry, I can't understand what you mean.

Comment (cstyan) [reply promql/test.go]: I don't think the `,` at the end of `... result in range mode,` is necessary. The previous comment for this function was different so maybe this one was incomplete or edited a few times but this was left in unintentionally. cc @brian-brazil

Comment (brian-brazil) [reply promql/test.go]: I think either way works

Comment (yuxiaobo96) [reply promql/test.go]: Ok, I have deleted this line of comments.

Comment (brian-brazil) [reply promql/test.go]: This comment should remain

Comment (yuxiaobo96) [reply promql/test.go]: Ok, I am going to modify it now.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**